Remove reference to issue 86

mmccool opened the pull request:
Resolves #86 

There was an old reference to issue #86 which basically said "we should use a Thing Model".  Well, we ARE using a Thing Model now, so I closed issue 86 and removed the reference to it in the text.  I also added some missing markup in the intro paragraph and tweaked the opening sentence again to make it clearer (I hope) that the TM applies to Thing Description Directories as a class.


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/mmccool/wot-discovery/pull/350.html" title="Last updated on Jun 20, 2022, 2:20 PM UTC (88d50f5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-discovery/350/a3ab94c...mmccool:88d50f5.html" title="Last updated on Jun 20, 2022, 2:20 PM UTC (88d50f5)">Diff</a>

farshidtz commented on index.html:
```diff
@@ -2379,26 +2379,14 @@ <h4>Semantic search: SPARQL</h4>
-                        The <a>TM</a> should describe all interaction affordances and indicate which ones
-                        are required (i.e. mandatory by this spec). Moreover, it should use placeholders
-                        for deployment-specific attributes.
-                        Informative affordances (JSONPath and XPath) should be defined as such.
```
The issue isn't fully resolved. In particular, we still have the informative JSONPath and XPath in there.

I suggest we simply remove them since the TM is normative. The text already covers everything that is necessary for implementations. We could also move them to another TM but that's really unnecessary.

mmccool replied:
We could also just add a comment to the TM, in the form of a "description", that the corresponding parts of the TM are not normative.

mmccool commented:
Updated.
- Text revised to make clearer that the TM IS normative, "except where noted", but that the rest of the specification in the section is applicable.
- Added notes in descriptions of searchJSONPath and searchXPath saying that "This affordance is not normative and is provided for information only."
- Added following sentence to end of intro text: "The searchJSONPath and searchXPath affordances given in this Thing Model are not normative and are provided for information only."
